libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Core/Arbitrary.hs

```diff
@@ -482,7 +483,7 @@ instance Arbitrary PoolParams where
       <*> arbitrary
 
 instance Arbitrary PoolMetadata where
-  arbitrary = PoolMetadata <$> arbitrary <*> arbitrary
+  arbitrary = PoolMetadata <$> arbitrary <*> (Data.ByteString.pack <$> vectorOf 32 (choose (0, 255)))
```

Comment:
This is a terrible way of generating byte strings. We have a helper [`genByteString`](https://github.com/IntersectMBO/cardano-ledger/blob/685f1da08d144e87c74f32a62bf16f9827fecff9/libs/cardano-ledger-binary/testlib/Test/Cardano/Ledger/Binary/Arbitrary.hs#L221-L222), which is already in scope, that is much more efficient: 
```suggestion
  arbitrary = PoolMetadata <$> arbitrary <*> genByteString 32
```